AdaSupport/documentation
# Web notifications doc

## What changed
added a section for web push notifications. 

## To test
- Pull
- `make serve`
- uncomment `push_notifications.md.erb` in `index.html.erb`
- you should be able to access the web docs for review
